### PR TITLE
Invalidate "bad" cache results in CaaS

### PR DESCRIPTION
**Pull request by dinvlad**

### CODE REVIEW

**Comment by dinvlad** [comment]
We'd like to know if these would have any undesired effects and if not, this is what @barkasn was looking for I think (we also wanted that, at some point, until coming up with a workaround).

**Comment by cjllanwarne** [comment]
Why you might want this:
* You've been deleting files and cromwell is wasting time failing on call cache copying

Why you might not want this:
* You're in a shared Cromwell and you're failing to copy somebody else's results because you don't have permission.

**Comment by dinvlad** [comment]
Afaik, for CaaS we don't have an issue with (2), since our pipelines don't reuse somebody else's results. @barkasn thoughts?

**Comment by cjllanwarne** [comment]
In other words: if everyone on CaaS runs workflows as the same user, or all of your users have access to all of each other's files, then you shouldn't have a problem with setting this value.

 If that's not true, then you risk invalidating somebody else's valid cache entry because it didn't happen to work for you.

File: configs/caas/cromwell.conf.ctmpl
Comment (cjllanwarne) [inline configs/caas/cromwell.conf.ctmpl]: Revert this. You didn't change the default.

**Review by cjllanwarne — CHANGES_REQUESTED** [review]
One small change, otherwise this is your call as the user as far as I'm concerned.

File: configs/caas/cromwell.conf.ctmpl
Comment (dinvlad) [reply configs/caas/cromwell.conf.ctmpl]: I changed this because https://cromwell.readthedocs.io/en/stable/Configuring/#call-caching says that the default is `false`.. Is that incorrect?

Comment (dinvlad) [reply configs/caas/cromwell.conf.ctmpl]: Woops, I misread it. Will change.

**Review by cjllanwarne — APPROVED** [review]

**Review by aednichols — APPROVED** [review]

**Comment by aednichols** [comment]
@dinvlad two thumbs, OK to merge when you wish

**Comment by cjllanwarne** [comment]
@dinvlad / @barkasn - just a ping to remind you that this is still here (and not yet merged...)

**Pull request merged**
